README.md

```diff
@@ -2091,6 +2091,37 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
     </details>
 
+* <a id='remove-blank-lines-between-chained-functions'></a>(<a href='#remove-blank-lines-between-chained-functions'>link</a>) **Remove blank lines between chained functions.** [![SwiftFormat: blanklinesbetweenchainedfunctions](https://img.shields.io/badge/SwiftFormat-blankLinesBetweenChainedFunctions-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md#blanklinesbetweenchainedfunctions)
+
+  <details>
+
+  #### Why?
+
+  Improves readability and maintainability, making it easier to see the sequence of functions that are applied to the object.
+
+  ```swift
+  // WRONG
+  let planets = ["Mercury", "Venus", "Earth", "Mars"]
+
+  var transformedPlanetNames: [String] {
+      planets
+        .map { $0.uppercased() } // Making names uppercase
+        
+        .map { $0 + "!" } // Adding excitement with an exclamation mark
+  }
+
+  // RIGHT
+  let planets = ["Mercury", "Venus", "Earth", "Mars"]
+
+  var transformedPlanetNames: [String] {
+      planets
+        .map { $0.uppercased() } // Making names uppercase
+        .map { $0 + "!" } // Adding excitement with an exclamation mark
+  }
+  ```
+
+  </details>
```

Comment:
> The rule does not remove this empty space if there is a comment in between chained functions. Running the linter will leave this code as-is.
>
> ```swift
> let planets = ["Mercury", "Venus", "Earth", "Mars"]
> 
> var transformedPlanetNames: [String] {
>    planets
>      .map { $0.uppercased() } // Making names uppercase
>
>      // Adding excitement with an exclamation mark
>      .map { $0 + "!" }
> }
>  ```
  
Is this exception necessary? I'd prefer we not allow this, and for consistency remove the blank line here as well.

Looking at the [PR](https://github.com/nicklockwood/SwiftFormat/pull/1480) that introduced this SwiftFormat rule, I don't see any indication that this behavior was implemented intentionally. It would be easy to update the SwiftFormat rule to handle this case and remove this blank line as well.

What do you think @mannylopez?

Reply:
Thanks for the help updating the rule. Here's the SwiftFormat PR to get rid of this exception: https://github.com/nicklockwood/SwiftFormat/pull/1723